### DISCUSSION TITLE
Error message when running the first tutorial notebook

### DISCUSSION THREAD

**Discussion by dyXDDD** [aertslab/SCENIC#277] [VSN] [answered]
Hello,

I had an issue when running the tutorial notebook SCENIC_tutorial_1-RunningVSN.ipynb. When I was running the last step for the entire pipeline, which was commented in the notebook, it gave the error message pasted at the end. It says the directory `/ddn1/vol1/staging/leuven/stg_00002/lcb/cflerin/resources` does not exist. Is it a directory in the container, or should I change it to a directory in my system?

In the previous preprocessing and filtering steps, I met similar issues, but solved this by changing the path in singularity.runOptions to directory in my system. But I don't know how to fix this in the entire pipeline. I have attached the config file. Could you please help me check what's wrong?

Thank you.

I changed the config file to .txt since GitHub doesn't support .config attachments.
[pbmc10k.vsn-pipelines.complete.txt](https://github.com/aertslab/SCENIC/files/6706388/pbmc10k.vsn-pipelines.complete.txt)

Error message:
=======================

executor >  local (12)
......
Error executing process > 'single_sample_scenic:SCENIC_APPEND:scenic:ARBORETO_WITH_MULTIPROCESSING (1)'

Caused by:
  Process `single_sample_scenic:SCENIC_APPEND:scenic:ARBORETO_WITH_MULTIPROCESSING (1)` terminated with an error exit status (255)

executor >  local (12)
......
[89/0c2612] process > single_sample_scenic:SCENIC_APPEND:scenic:ARBORETO_WITH_MULTIPROCESSING (1)                        [100%] 1 of 1, failed: 1 ✘
......
Error executing process > 'single_sample_scenic:SCENIC_APPEND:scenic:ARBORETO_WITH_MULTIPROCESSING (1)'

Caused by:
  Process `single_sample_scenic:SCENIC_APPEND:scenic:ARBORETO_WITH_MULTIPROCESSING (1)` terminated with an error exit status (255)

Command executed:

  arboreto_with_multiprocessing.py             pbmc10k_data.filtered.loom             allTFs_hg38.txt             --output pbmc10k_data__adj.tsv.gz             --num_workers 4             --cell_id_attribute CellID             --gene_attribute Gene             --method grnboost2             --seed 250

Command exit status:
  255

Command output:
  (empty)

Command error:
  WARNING: skipping mount of /ddn1/vol1/staging/leuven/stg_00002/lcb/cflerin/resources: stat /ddn1/vol1/staging/leuven/stg_00002/lcb/cflerin/resources: no such file or directory
  FATAL:   container creation failed: mount /ddn1/vol1/staging/leuven/stg_00002/lcb/cflerin/resources->/ddn1/vol1/staging/leuven/stg_00002/lcb/cflerin/resources error: while mounting /ddn1/vol1/staging/leuven/stg_00002/lcb/cflerin/resources: mount source /ddn1/vol1/staging/leuven/stg_00002/lcb/cflerin/resources doesn't exist

Work dir:
  /gpfs/slayman/pi/gerstein/yd283/tutorials/SCENIC_pbmc/work/89/0c26127c8bdaf6de46c7ce5004fc36

**Comment by gianfilippo** [comment]
same here using nextflow

**Comment by KrisDavie** [comment, marked as answer]
This error message is because you have not changed the paths to the one/all of following parameters:

- `scenic.grn.tfs`
- `scenic.cistarget.motifsDb`
- `scenic.cistarget.motifsAnnotation`
- `scenic.cistarget.tracksDb`
- `scenic.cistarget.tracksAnnotation`
 
You will need to have these files available in your local computing platform and put the correct paths in these config parameters.
